index.html

```diff
@@ -1,304 +1,243 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="styles.css" />
-    <link
-      href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet"
-    />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">      
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
   </head>
   <body>
-    <div class="main-container">
-      <nav class="nav-bars">
-        <div class="logo">
-          <a href="index.html"><h2>Duncan</h2></a>
+      <div id="header">
+      <header id="navigation-bar">
+        <div id="logo">
+          <a href="#header"><img src="img/MyLogo.png" alt="Logo"></a>
         </div>
-        <nav class="menu">
-          <nav><img src="photo/navs/Union.png" alt="" /></nav>
+        <nav id="nav">
+          <div id="btn-menu"><img src="img/Normal-Button/Tertiary/Icons/Icon-Menu.png" alt=""></div>
+          <ul id="menu-lst">
+            <li><a href="#headline" class="menu-lst-txt">Portfolio</a></li>
+            <li><a class="menu-lst-txt" href="#About-my-self">About</a></li>
+            <li><a class="menu-lst-txt" href="#Contact-Section">Contact</a></li>
+          </ul>
         </nav>
-      </nav>
-
-      <section class="About-me">
-        <div class="greeting">
-          <h1>I'm Duncan Glad to see you!</h1>
-        </div>
-
-        <div class="biography">
-          <p>
-            I’m a software developer! I can help you design and build a product
-            , feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
+      </header>
+      <section id="headline">
+        <div class="headline-txt">
+          <h1 class="txt-1 order-0 pd-0 mrg-0 mbl-to-hide h1-animation">I’m Duncan Glad to see you!</h1>
+          <h1 class="txt-1-dsktp order-0 pd-0 mrg-0 dsktp-to-hide dsktp-to-show h1-animation">I’m Duncan <br>
+            Glad to see you!</h1>
+          <p class="txt-2-dsktp order-1 pd-0 mrg-0 txt-j">I’m a software developer! I can help you build
```

Comment:
- [ ] I can see that you have so many vertical scroll bars on almost every part of your heading section. Please ensure you remove them as they not making your UI look so good 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/187036693-90b2f224-05f6-4b63-9f5f-c1afbe3bf617.png)

![image](https://user-images.githubusercontent.com/63473584/187036726-7c46515b-9fd4-4898-8228-03b0722b49cb.png)